geode-core/src/distributedTest/java/org/apache/geode/cache30/RegionReliabilityTestCase.java

```diff
@@ -1341,7 +1341,7 @@ public void run() {
       // pass
     }
 
-    // CLEAR
+    // CLEAR_REGION
```

Comment:
I think that several of these renames were accidentally included when changing the Cli string. They should be put back how they were.

Reply:
Yep, this was a careless rename on my part. I changed these back.